.eslintrc

```diff
@@ -24,6 +24,7 @@
     }
   },
   "rules": {
+    "backpack/use-components": "off",
```

Comment:
Presumably this is off because we can't use a backpack component inside a backpack component?

Reply:
Yeah, there are legit places where we need to use HTML elements, for example we use `<code />` inside `<BpkCode />`. There were over 100 errors like this. @olliecurtis and I discussed it and decided this brute force solution was better because we already know to use our own components, and it saves the tedium of adding over 100 opt outs.

Reply:
Yeah I guess if we haven't used backpack there's probably a very good reason for it 😉